### PR TITLE
Feat/forgot abha number

### PR DESCRIPTION
**Pull request by yashasva01**

### CODE REVIEW

File: src/routes/retrieveAbhaRoutes.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const router = require('express').Router()
+const validator = require('../util/middleware/validator')
+const retrieveABHAController = require('../controllers/retrieveAbhaController')
+
+router.post('/existsByHealthId',
+  validator.validate(validator.schemas.checkAbhaexistsService, validator.REQ_PARAMTERS.body),
```
Comment (souris-dev) [inline src/routes/retrieveAbhaRoutes.js:6]: nit: Let's use camel case - `checkAbhaExistsService`

File: src/services/retrieveAbhaServices.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+const axios = require('axios')
+const adapterUrl = require('../../config')
+
+async function checkAbhaexistsService (healthId) {
```
Comment (souris-dev) [inline src/services/retrieveAbhaServices.js:4]: nit: Let's use camel case - checkAbhaExistsService

File: src/util/middleware/validator.js [hunk 3]
```diff
@@ -41,6 +41,22 @@ const schemas = {
     otp: joi.string().required(),
     authMethod: joi.string().valid('AADHAAR_OTP', 'MOBILE_OTP').required()
   }),
+
+  checkAbhaexistsService: joi.object({
```
Comment (souris-dev) [inline src/util/middleware/validator.js:45]: nit: Let's use camel case - checkAbhaExistsService

**Review by souris-dev — CHANGES_REQUESTED** [review]

**Review by souris-dev — APPROVED** [review]

File: src/routes/retrieveAbhaRoutes.js (hunk 1 above)
Comment (yashasva01) [reply src/routes/retrieveAbhaRoutes.js]: done

File: src/services/retrieveAbhaServices.js (hunk 2 above)
Comment (yashasva01) [reply src/services/retrieveAbhaServices.js]: done

**Review by SaatwikRishi — APPROVED** [review]

**Pull request merged**